ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/platform_settings.schema.yml

```diff
@@ -98,14 +98,35 @@ keys:
           non_mlag: 330
       - platforms:
           - 7050X3
+        feature_support:
+          queue_monitor_length_notify: false
+        reload_delay:
+          mlag: 300
+          non_mlag: 330
+        trident_forwarding_table_partition: flexible exact-match 16384 l2-shared 98304 l3-shared 131072
+      - platforms:
           - 720XP
-          - 722XP
         feature_support:
+          poe: true
           queue_monitor_length_notify: false
         reload_delay:
           mlag: 300
           non_mlag: 330
         trident_forwarding_table_partition: flexible exact-match 16384 l2-shared 98304 l3-shared 131072
+      - platforms:
+          - '750'
+          - '755'
+          - '758'
+          - 720D
```

Comment:
Same comment as above regarding `720D` needing to be changed to `720DP`

Reply:
I agree with @jrecchia1029 - see platform specification: https://www.arista.com/en/products/720d-series/specifications

Reply:
Thanks.  Will update this shortly.